variables.tf

```diff
@@ -177,6 +177,32 @@ variable "jira_db_iops" {
   type        = number
 }
 
+variable "jira_db_snapshot_identifier" {
+  description = "The identifier for the DB snapshot to restore from. The snapshot should be in the same AWS Region as the DB instance."
+  default     = null
+  type        = string
+}
+
+variable "jira_rds_master_username" {
+  description = "Master username for the Jira RDS instance."
+  type        = string
+  default     = null
```

Comment:
Can we make the `default = postgres` so it is explicitly defined here and easier to find? We would also save a conditional check in `locals.tf`.

Reply:
We can, but I this default is for postgres. This is true that postgres is the only supported DB but if we introduce any other database in future, they will have different defaults. So I moved the default to the RDS module instead. In config file is documented about default if username is not provided.